Include/internal/pycore_frame.h

```diff
@@ -65,6 +65,7 @@ typedef struct _PyInterpreterFrame {
     _Py_CODEUNIT *instr_ptr; /* Instruction currently executing (or about to begin) */
     int stacktop;  /* Offset of TOS from localsplus  */
     uint16_t return_offset;  /* Only relevant during a function call */
+    uint16_t tier2_extra_size; /* How many extra entries is at the end of localsplus for tier 2 inlining */
```

Comment:
These extra 2 bytes will cost 8 bytes due to alignment. Could it be one byte? (Or better, could this be somehow calculated statically and put in the trace instead? So we only pay for it when actually using it.)

Reply:
We could just block inlining unless the current stack has enough stack space. That should mean we never need `_GROW_TIER2_STACK` or similar.